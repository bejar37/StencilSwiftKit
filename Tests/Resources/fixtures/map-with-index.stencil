// map items1
{% map items1 with item set mapped1 %}
	- #{{maploop.counter}}: {{item}} (first: {{maploop.first}}, last: {{maploop.last}})
{% endmap %}

// map items2
{% map items2 with item set mapped2 %}{{item}}{% if maploop.last %}!{% else %} {% endif %}{% endmap %}

Value of mapped1:
{% for item in mapped1 %}
{{item}}
{% endfor %}

Value of mapped2:
{% for item in mapped2 %}{{item}}{% endfor %}
