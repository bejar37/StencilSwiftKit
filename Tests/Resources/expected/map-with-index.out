// map items1

// map items2

Value of mapped1:
	- #0: one (first: true, last: false)
	- #1: two (first: false, last: false)
	- #2: three (first: false, last: true)

Value of mapped2:
hello world everyone!
